tests/Symfony/Upgrade/Test/Fixer/FormGetnameToGetblockprefixFixerTest.php

```diff
@@ -12,10 +12,48 @@ public function testFix($expected, $input, $file)
         $this->makeTest($expected, $input, $file);
     }
 
+    /**
+     * @dataProvider provideFailingExamples
+     */
+    public function testWrongFix($expected, $input, $file, $unExpected)
+    {
+        $fixedCode = $this->getFixer()->fix($file, $input);
+        $this->assertNotSame($unExpected, $fixedCode);
+        $this->assertSame($expected, $fixedCode);
+    }
+
     public function provideExamples()
```

Comment:
Because the `makeTest` function doesn't work when input and output should be the same. I had to work around that.